docs/source/arguments.rst

```diff
@@ -304,3 +303,69 @@ dglke_partition
     The number of partitions.
 
+dglke_score
+^^^^^^^^^^^^^^^
+  - ``--data_path DATA_PATH``
+    The root path of all dataset including id mapping files. Default: 'data'
+
+  - ``--model_path MODEL_PATH``
+    The place where to load the model. Default 'ckpts'.
+
+  - ``--format FORMAT``
+    The format of the input data, specified in ``h_r_t``. Ideally, user should provides three files, one for head entities, one for relations and one for tail entities. But we also allow users to use *** to represent *all* of the entities or relations. For example, ``h_r_*`` requires users to provide files containing head entities and relation entities and use the whole entity set as tail entities; ``*_*_t`` requires users to provide a single file containing tail entities and use the whole entity set as head entities and the whole relation set as relations. The supported formats include ``h_r_t``, ``h_r_*``, ``h_*_t``, ``*_r_t``, ``h_*_*``, ``*_r_*``, ``*_*_t``. By default, the calculation will take an N_h x N_r x N_t manner.
+
+  - ``--data_files [DATA_FILES ...]`` 
+    A list of data file names. This is used to provide necessary files containing the requried data according to the format, e.g., for ``h_r_t``, three files are required as h_data, r_data and t_data, while for ``h_*_t``, two files are required as h_data and t_data.
+
+  - ``--raw_data``
+    A flag tells whether the data profiled in data_files is in the raw object naming space or in mapped id space. If True, the data is in the original naming space and the inference program will do the id translation according to id mapping files. If False, the data is just intergers and it is assumed that user has already done the id translation. Default: False.
+
+  - ``--bcast``
+    Whether to broadcast topK in a specific side. By default, an universal topK across all scores are returned. Users can specify ``head`` to broadcast at head that returns topK for each head; ``rel`` to broadcast at relation that returns topK for each relation; ``tail`` to broadcast at tail that returns topK for each tail. Default: None.
+
+  - ``--topk NUM_OF_K``
+    How many results are returned. Default:10.
+
+  - ``--score_func SCORE_FUNC_NAME or None``
+    What kind of score is used in ranking. Currently, we support two functions: ``none`` (score = $x$) and ``logsigmoid`` ($score = log(sigmoid(x))$). Default: none.
+
+  - ``--gpu GPU_ID``
+    GPU device to use in inference, by default it uses CPU. Default: -1.(CPU)
+
+  - ``--output FILE_PATH``
+    Where to store the result. Default: result.tsv
+
+  - ``--entity_mfile`` (Optional)
+    The entity ID mapping file. If not provided we will use the mapping file in ``--data_path`` according to the config.json under ``--model_path``.
+
+  - ``--rel_mfile`` (Optional)
+    The relation ID mapping file. If not provided we will use the mapping file in ``--data_path`` according to the config.json under ``--model_path``.
+
+dglke_emb_sim
```

Comment:
if it does what i think, i feel this should be called `dglke_score`.

Reply:
Score for $score = f(h, r, t)$
emb_sim for entity embedding similarity or relation embedding similarity.
Any good idea on the naming of the CMD?

Reply:
i think i understand it now.